### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[X] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Review by shamsher31 — APPROVED** [review]
Thanks

**Review by sdake — APPROVED** [review]
LGTM.

File: samples/addons/kiali.yaml [hunk 1]
```diff
@@ -874,7 +874,7 @@ spec:
       - label: "id"
         displayName: "Space"
   - chart:
-      name: "Memory commited"
+      name: "Memory committed"
```
Comment (howardjohn) [inline samples/addons/kiali.yaml:877]: This comes from https://kiali.org/helm-charts, we should not modify it here

**Review by esnible — APPROVED** [review]

**Pull request merged**